test_app.py

```diff
@@ -8,5 +11,14 @@ def test_dummy(self):
     def test_ci_workflow(self):
         self.assertEqual(True, True)
 
+class TestApplication(unittest.TestCase):
+
+    def test_welcome_message(self):
+        captured_output = StringIO()
+        sys.stdout = captured_output
+        Info.welcome_message()
+        output = captured_output.getvalue().strip()
+        self.assertEqual(output, 'Welcome to Tic Tac Toe')
```

Comment:
I know you're reading the Beck TDD book at the moment - this might be a good case to apply the section on removing duplication. As in, the 'Welcome to Tic Tac Toe' string is duplicated both here and in `tictactoe.py` - is it possible to remove the duplication?